<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewMLPDialog</class>
 <widget class="QDialog" name="addNewMLPDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Basico</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Entradas</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbInputSize">
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>403</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Salidas</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbOutputSize">
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>36</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Capas</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tblLayers">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>74</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>27</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QToolButton" name="btnAddLayer">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../imagenes.qrc">
              <normaloff>:/imagenes/plus_icon.png</normaloff>:/imagenes/plus_icon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnDeleteLayer">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../imagenes.qrc">
              <normaloff>:/imagenes/minus_icon.png</normaloff>:/imagenes/minus_icon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Funcion de transferencia</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="cbTrasnferFunction">
          <item>
           <property name="text">
            <string>Sigmoide</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tangente</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Pendiente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbSlope">
          <property name="maximum">
           <number>9999999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
